Travis failures

There are a couple of [travis failures](https://api.travis-ci.org/v3/job/517027366/log.txt) in `master` on the RDKit testing branch that should be easy to fix:
```
=================================== FAILURES ===================================
_ [doctest] openforcefield.topology.molecule.FrozenMolecule.compute_partial_charges _
2013             The partial charge calculation method to use for partial charge calculation.
2014         toolkit_registry : openforcefield.utils.toolkits.ToolRegistry or openforcefield.utils.toolkits.ToolkitWrapper, optional, default=None
2015             :class:`ToolkitRegistry` or :class:`ToolkitWrapper` to use for SMILES-to-molecule conversion
2016 
2017         Examples
2018         --------
2019 
2020         >>> molecule = Molecule.from_smiles('CCCCCC')
2021         >>> molecule.generate_conformers()
2022         >>> molecule.compute_partial_charges()
UNEXPECTED EXCEPTION: NotImplementedError()
Traceback (most recent call last):

  File "/home/travis/miniconda/envs/test/lib/python3.6/doctest.py", line 1330, in __run
    compileflags, 1), test.globs)

  File "<doctest openforcefield.topology.molecule.FrozenMolecule.compute_partial_charges[2]>", line 1, in <module>

  File "/home/travis/build/openforcefield/openforcefield/openforcefield/topology/molecule.py", line 2030, in compute_partial_charges
    raise NotImplementedError

NotImplementedError

/home/travis/build/openforcefield/openforcefield/openforcefield/topology/molecule.py:2022: UnexpectedException
_ [doctest] openforcefield.typing.engines.smirnoff.forcefield.ForceField.__init__ _
211         >>> forcefield = ForceField('smirnoff99Frosst.offxml')
212 
213         Load multiple SMIRNOFF parameter sets:
214 
215         forcefield = ForceField('smirnoff99Frosst.offxml', 'tip3p.offxml')
216 
217         Load a parameter set from a string:
218 
219         >>> offxml = '<SMIRNOFF version="1.0" aromaticity_model="OEAroModel_MDL"/>'
220         >>> forcefield = ForceField(offxml)
UNEXPECTED EXCEPTION: SMIRNOFFVersionError(SMIRNOFFVersionError(...), 'SMIRNOFF offxml file was written with version 1.0, but this version of ForceField only supports up to version 0.2')
Traceback (most recent call last):

  File "/home/travis/miniconda/envs/test/lib/python3.6/doctest.py", line 1330, in __run
    compileflags, 1), test.globs)

  File "<doctest openforcefield.typing.engines.smirnoff.forcefield.ForceField.__init__[2]>", line 1, in <module>

  File "/home/travis/build/openforcefield/openforcefield/openforcefield/typing/engines/smirnoff/forcefield.py", line 245, in __init__
    self.parse_sources(sources)

  File "/home/travis/build/openforcefield/openforcefield/openforcefield/typing/engines/smirnoff/forcefield.py", line 613, in parse_sources
    self.load_smirnoff_data(smirnoff_data)

  File "/home/travis/build/openforcefield/openforcefield/openforcefield/typing/engines/smirnoff/forcefield.py", line 674, in load_smirnoff_data
    self._check_smirnoff_version_compatibility(str(version))

  File "/home/travis/build/openforcefield/openforcefield/openforcefield/typing/engines/smirnoff/forcefield.py", line 283, in _check_smirnoff_version_compatibility
    'up to version {}'.format(version, self._MAX_SUPPORTED_SMIRNOFF_VERSION))

openforcefield.typing.engines.smirnoff.forcefield.SMIRNOFFVersionError: (SMIRNOFFVersionError(...), 'SMIRNOFF offxml file was written with version 1.0, but this version of ForceField only supports up to version 0.2')

/home/travis/build/openforcefield/openforcefield/openforcefield/typing/engines/smirnoff/forcefield.py:220: UnexpectedException
=============================== warnings summary ===============================
versioneer.py:941
  /home/travis/build/openforcefield/openforcefield/versioneer.py:941: DeprecationWarning: invalid escape sequence \s
    '''

openforcefield/typing/chemistry/environment.py:53
  /home/travis/build/openforcefield/openforcefield/openforcefield/typing/chemistry/environment.py:53: DeprecationWarning: invalid escape sequence \(
    """

examples/SMIRNOFF_simulation/run_simulation.ipynb::Cell 0
  /home/travis/miniconda/envs/test/lib/python3.6/site-packages/jupyter_client/manager.py:62: DeprecationWarning: KernelManager._kernel_spec_manager_changed is deprecated in traitlets 4.1: use @observe and @unobserve instead.
    def _kernel_spec_manager_changed(self):

openforcefield/tests/test_forcefield.py::TestForceFieldParameterAssignment::test_alkethoh_parameters_assignment[r0]
  /home/travis/miniconda/envs/test/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.ufunc size changed, may indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
    return f(*args, **kwds)
```

--- a/openforcefield/topology/molecule.py
+++ b/openforcefield/topology/molecule.py
@@ -2019,7 +2019,6 @@ class FrozenMolecule(Serializable):
 
         >>> molecule = Molecule.from_smiles('CCCCCC')
         >>> molecule.generate_conformers()
-        >>> molecule.compute_partial_charges()
 
         Raises
         ------

--- a/openforcefield/typing/engines/smirnoff/forcefield.py
+++ b/openforcefield/typing/engines/smirnoff/forcefield.py
@@ -216,7 +216,7 @@ class ForceField(object):
 
         Load a parameter set from a string:
 
-        >>> offxml = '<SMIRNOFF version="1.0" aromaticity_model="OEAroModel_MDL"/>'
+        >>> offxml = '<SMIRNOFF version="0.2" aromaticity_model="OEAroModel_MDL"/>'
         >>> forcefield = ForceField(offxml)
 
         """

